First_section.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <section>
+        <header class="toolbar">
+          <h1 id="Logo"><a href="." rel="home">My Logo</a></h1>
+          <nav><img id="menu" src="union.png" /></nav>
+        </header>
+      </section>
+
+      <main class="content">
+        <span class="text_header">
+          <p id="welcome">I am Christian Glad to see you!</p>
```

Comment:
[Optional]

- [ ] This is required but since it was missed by the first reviewer I'm marking it as optional. Please consider making this an `h1` tag since it's the main header of the page 👍